src/cryptoadvance/specter/controller.py

```diff
@@ -143,22 +193,31 @@ def settings():
     host = rpc['host']
     protocol = 'http'
     explorer = app.specter.explorer
-    auth = app.specter.config["auth"]
-    hwi_bridge_url = app.specter.config['hwi_bridge_url']
+    auth = app.specter.config['auth']
```

Comment:
Maybe add here:
```py
    if auth=='none':
        app.login('admin')
```
Worked for me...

Reply:
Oh I see why that's needed, thanks, added.